nemo/collections/tts/modules/encodec_modules.py

```diff
@@ -59,13 +60,33 @@
 from nemo.utils.decorators import experimental
 
 
+class CodecActivation(nn.Module):
```

Comment:
Can we move this to `audio_codec_modules.py`? I think we should avoid adding code to `encodec_modules.py` that is not part of the original encodec source.

Reply:
Sure, good point.